src/psyclone/gocean1p0.py

```diff
@@ -889,14 +1106,17 @@ class GO1p0Descriptor(Descriptor):
     def __init__(self, kernel_name, kernel_arg):
 
```

Comment:
Please document the interface to this routine.

Reply:
Done.